.github/workflows/crypto_secretstream.yml

```diff
@@ -0,0 +1,72 @@
+name: crypto_secretstream
+
+on:
+  pull_request:
+    paths:
+      - "crypto_secretstream/**"
+      - "Cargo.*"
+  push:
+    branches: master
+
+defaults:
+  run:
+    working-directory: crypto_secretstream
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+
+jobs:
+  #  # TODO: test no_std builds when rust-lang/cargo#7916 is on stable
+  #  build:
+  #    runs-on: ubuntu-latest
+  #    strategy:
+  #      matrix:
+  #        rust:
+  #          - 1.49.0 # MSRV
+  #          - stable
+  #        target:
+  #          - thumbv7em-none-eabi
+  #          - wasm32-unknown-unknown
+  #    steps:
+  #      - uses: actions/checkout@v1
+  #      - uses: actions-rs/toolchain@v1
+  #        with:
+  #          profile: minimal
+  #          toolchain: ${{ matrix.rust }}
+  #          target: ${{ matrix.target }}
+  #          override: true
+  #      - run: cargo build --no-default-features --release --target ${{ matrix.target }}
```

Comment:
FYI, I think this could work in this particular repo since there may not be dependencies that activate `std` features. Maybe give it a try?